engine/runtime/src/main/scala/org/enso/interpreter/instrument/command/DeserializeLibrarySuggestionsCmd.scala

```diff
@@ -19,8 +19,10 @@ class DeserializeLibrarySuggestionsCmd(
   override def execute(implicit
     ctx: RuntimeContext,
     ec: ExecutionContext
-  ): Future[Unit] =
+  ): Future[Unit] = {
     ctx.jobProcessor.runBackground(
       new DeserializeLibrarySuggestionsJob(request.libraryName)
     )
+    Future.successful(())
```

Comment:
Is the operation `successful` when the job is just submitted? Shouldn't `runBackground` return a `Future` to track how "successful" the operation is?

Reply:
`runBackground` just schedules the job execution and does not imply that the job will be started (or even submitted) for execution right away.